token/js/src/extensions/index.ts

```diff
@@ -2,6 +2,8 @@ export * from './accountType.js';
 export * from './cpiGuard/index.js';
 export * from './defaultAccountState/index.js';
 export * from './extensionType.js';
+export * from './group/index.js';
+export * from './groupMember/index.js';
```

Comment:
What do you think about just combining the two directories `group` and `groupMember` into one directory `tokenGroup`? That way everything for the Token Group interface is all in one place.

Reply:
Sure, I put them in different dir coz I saw these are 2 extensions

Reply:
Makes sense, but I'd lean on combining them if it's all the same to you!